main.py

```diff
@@ -0,0 +1,182 @@
+import asyncio
+import os
+import time
+from datetime import datetime as dt
+from random import randint
+
+import aioredis
+from aiogram import Bot, Dispatcher
+from aiogram.contrib.fsm_storage.memory import MemoryStorage
+from aiogram.dispatcher.filters.state import State, StatesGroup
+from aiogram.utils import executor
+from dotenv import load_dotenv
+
+import constants
+
+load_dotenv()
+
+TELEGRAM_TOKEN = os.environ.get('TELEGRAM_TOKEN')
+if TELEGRAM_TOKEN is None:
+    raise SystemExit('Не удалось получить токены')
+
+
+bot = Bot(token=TELEGRAM_TOKEN)
+storage = MemoryStorage()
+dp = Dispatcher(bot=bot, storage=storage)
+
+
+class Form(StatesGroup):
+    name = State()
+    birthday = State()
+    yes_or_not = State()
+
+
+@dp.message_handler(commands='start')
+async def cmd_start(message):
+    """
+    Точка начала диалога.
+    """
+    await Form.name.set()
+
+    await message.reply('Привет! Как тебя зовут?')
+
+
+@dp.message_handler(state=Form.name)
+async def process_name(message, state):
+    """
+    Установка имени пользователя.
+    :param message:
+    :param state:
+    :return:
+    """
+    async with state.proxy() as data:
+        data['name'] = message.text
+        data['chat_id'] = message.chat.id
+    await Form.next()
+    await message.reply('Введи дату своего рождения в формате "ДД-ММ-ГГ"')
+
+
+@dp.message_handler(state=Form.birthday)
+async def process_birthday(message, state):
+    """
+    Установка дня рождения.
+    :param message:
+    :param state:
+    :return:
+    """
+    birthday = dt.strptime(message.text, '%d-%m-%y')
+    async with state.proxy() as data:
+        data['birthday'] = birthday
+        data['congratulations'] = False
+    await Form.next()
+    await message.reply('Введи любое сообщение что бы начать!')
+
+
+@dp.message_handler(state=Form.yes_or_not)
+async def message_loop(message, state):
+    """
+    Основной цикл бота.
+    :param message:
+    :param state:
+    :return:
+    """
+    await message.reply('С этого момента ты будешь получать сообщения!')
+
+    redis_connection = await aioredis.create_connection("redis://localhost")
+    redis = await aioredis.Redis(pool_or_conn=redis_connection)
+
+    async with state.proxy() as data:
+        chat_id = data['chat_id']
+        birthday = data['birthday']
+
+    current_state = await state.get_state()
+
+    while current_state is not None:
+        current_state = await state.get_state()
+        date_today = dt.now()
+        time_now = date_today.time()
+        table = f'user:list:{chat_id}'
```

Comment:
Better use fucntion. Like def id_table(chat_id): return f'user:list:{chat_id}'.
And better keep it in constants.